scripts/db_perf.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
Could you add some more comments to this file: what its for and what it does.

Reply:
done